README.md

```diff
@@ -30,7 +30,7 @@ Install dependencies:
 $ npm install
 ```
 
-Rock and Roll
+Start the app:
```

Comment:
Please use a different PR for this change, rather than bundling with an unrelated change.